saleor/urls.py

```diff
@@ -25,6 +25,7 @@
     url(r'^checkout/', include(checkout_urls, namespace='checkout')),
     url(r'^dashboard/', include(dashboard_urls, namespace='dashboard')),
     url(r'^graphql', GraphQLView.as_view(graphiql=settings.DEBUG)),
+    url(r'^impersonate/', include('impersonate.urls')),
```

Comment:
I think urls file is missing.

Reply:
`impersonate` is external app, this resolves nicely

Reply:
Fine then.